gatsby-node.js

```diff
@@ -331,6 +334,7 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
       path: program.fields.slug,
       component: ProgramPostTemplate,
       context: {
+        osProgram: program.frontmatter.program,
```

Comment:
better to name it `program` than `osProgram`

Reply:
Done